plugins/koa/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@reframe/koa",
+  "version": "0.0.1",
+  "dependencies": {
+    "@brillout/cli-theme": "0.1.5",
+    "@brillout/repage": "0.1.5",
```

Comment:
We don't need @brillout/repage and @reframe/utils anymore. But we need @brillout/reconfig now

Reply:
Done